tox.ini

```diff
@@ -1,19 +1,14 @@
 [tox]
-envlist = py{27,35}-django111-drf{39,latest}, py35-django{20,21,22}-drf{latest}, quality
-
+envlist = py35-django{22,30}, quality
```

Comment:
Django 3.0 dropped support for Python 3.5, so that combination wouldn't work.

And also we have to add the py38  env